catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -194,6 +195,23 @@ define([
                         });
                         wreqr.vent.trigger('search:polydisplay', this.model);
                         break;
+                    // these cases are for when the model matches the location model
```

Comment:
❓ @andrewkfiedler why did this change?

Reply:
Oh, that's a good question.  Basically we used to lose the location information when the user switched attributes, say from anyGeo to location.  Now we keep it around and use it!  So if you draw a circle and decide you want to search over a different geo attribute, you don't have to redraw.

Reply:
nice thanks